package.json

```diff
@@ -15,8 +15,8 @@
     "@chakra-ui/react": "^2.4.6",
     "@emotion/react": "^11",
     "@emotion/styled": "^11",
-    "@railgun-community/quickstart": "3.8.12",
-    "@railgun-community/shared-models": "3.7.6",
+    "@railgun-community/quickstart": "3.11.6",
```

Comment:
_(arbitrary comment location to start a thread)_

If I add `console.log('loading...')` as the first line in `loadProviders` in our `raulgun.ts` method, I still see it being called 4 times

<img width="1015" alt="image" src="https://user-images.githubusercontent.com/17163988/218633711-84619617-b740-4941-8898-8aa4e81bb449.png">

Reply:
That may be an artifact of it running locally, in strict mode. I tested with a console log on the production build and it only showed up once for me. I will push up a console log so we can test.

Reply:
Ah nvm, I was confused and thought we were targeting the initialize

Reply:
Seems to be working now, I left a log in to confirm

Reply:
I'm seeing it logged twice now, but that's probably fine since it shouldn't be an issue anymore anyway. Let's go with this and see how it goes
<img width="391" alt="image" src="https://user-images.githubusercontent.com/17163988/219106894-f8cb4c51-9ca6-4110-968b-4ece30339325.png">